jac-uk/admin
# include other qualfications

## What's included?
Custom 'Other' qualifications from admin were not pulling thru and showing on apply.
They were equally not showing upon admin edit of applications either.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Using a legal exercise on admin with a custom "other" qualification
- Edit an applications qualifications and ensure this custom qualification appears as an enterable option

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
